src/main/java/frc/robot/util/Constants.java

```diff
@@ -50,4 +50,7 @@
 
 	//kWristAcceleration - max acceleration of wrist motor in encoder units per 100 ms per 1 s
 	public static int kWristAcceleration = 5000;
+
+	//kRollerIRThreshold - threshold IR value when the cargo is in the rollers
```

Comment:
It's also useful to put the range of values if it's not intuitive (so in this case, I believe it is from 0-4096 since it's a 12-bit Analog-to-Digital Converter or ADC for short)